src/api.ts

```diff
@@ -0,0 +1,15 @@
+const API_URL = 'https://jsonplaceholder.typicode.com';
+
+const getData = async(url: string) => {
+  const response = await fetch(API_URL + url);
+
+  if (response.ok) {
+    return response.json();
+  }
+
+  return [];
+};
+
+export const getPosts = () => getData('/posts');
+export const getUsers = () => getData('/users');
+export const getComments = () => getData('/comments');
```

Comment:
@ivadimko
 But this file is not typed at all, It is hard to control if everything works as expected
Why shouldn't we force to add types here?